lib/plugins/wallet/geth/base.js

```diff
@@ -125,11 +124,10 @@ function generateTx (_toAddress, wallet, amount, includesFee, cryptoCode) {
   ]
 
   return Promise.all(promises)
-    .then(([gas, gasPrice, txCount, { baseFeePerGas }]) => [
```

Comment:
`baseFeePerGas` is removed here but there are more EIP-1559 leftovers. Should we leave it as is and handle this later?